test/components/structures/TimelinePanel-test.tsx

```diff
@@ -513,4 +516,64 @@ describe("TimelinePanel", () => {
             replyToEvent.mockClear();
         });
     });
+
+    it("renders when the last message is an undecryptable thread root", () => {
+        jest.spyOn(SettingsStore, "getValue").mockImplementation((name) => name === "feature_threadstable");
+
+        const client = MatrixClientPeg.get();
+        client.isRoomEncrypted = () => true;
+        client.supportsExperimentalThreads = () => true;
+        client.decryptEventIfNeeded = () => Promise.resolve();
+        const authorId = client.getUserId()!;
+        const room = new Room("roomId", client, authorId, {
+            lazyLoadMembers: false,
+        });
+
+        const events = mockEvents(room);
+        const props = {
+            ...getProps(room, events),
+        };
```

Comment:
why not just

```suggestion
        const props = getProps(room, events);
```

?